buildpacks/ruby/src/bin/agentmon_loop.rs

```diff
@@ -0,0 +1,204 @@
+// Enable Clippy lints that are disabled by default.
+// https://rust-lang.github.io/rust-clippy/stable/index.html
+#![warn(clippy::pedantic)]
+
```

Comment:
(Informational)

I was curious how large these binaries were, so checked the release builds:

```
$ du -ah
2.6M    ./agentmon_loop
2.6M    ./launch_daemon
5.1M    .
```

I suspect much of that comes from Clap (and some of its default feature: https://docs.rs/clap/latest/clap/_features/index.html#default-features).

Though compared to the size of the Ruby runtime, Ruby dependencies and app source for the typical Ruby app, it's a rounding error, so nothing to worry about.